### PR TITLE
fix: add conditional rendering of OAuth providers

### PR DESCRIPTION
**Pull request by anuragts**
Now google OAuth provider is not rendered if client id is not provided
Screenshot from 2023-12-06 17-39-09

fixes - #735

### CODE REVIEW

**Comment by Mythie** [comment]
Hey there, it looks like you haven't accepted our contributor license agreement yet. In order for us to accept your pull request we ask that you please fill out the CLA:

https://documen.so/cla

**Comment by anuragts** [comment]
Done !!

**Review by dephraiim — CHANGES_REQUESTED** [review]

File: apps/web/src/components/forms/signin.tsx [hunk 1]
```diff
@@ -30,6 +30,9 @@ const TwoFactorEnabledErrorCode = ErrorCode.TWO_FACTOR_MISSING_CREDENTIALS;
 
 const LOGIN_REDIRECT_PATH = '/documents';
 
+const shouldRender =
```
[the remark this answers is not in the document]
Comment (anuragts) [reply apps/web/src/components/forms/signin.tsx:33]: yo Duncan ,
yes I also though of this , will push necessary changes .

[the remark this answers is not in the document]
Comment (anuragts) [reply apps/web/src/components/forms/signin.tsx]: Renamed the variable to ALLOW_GOOGLE_OAUTH

**Review by dguyen — APPROVED** [review]
Hey! Apologies for the delay

I've pushed some minor changes since we can't actually access private environment variables on the client side.

Other than that, it's good, thanks a lot for this!

**Pull request merged**